core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -236,15 +236,6 @@ func getExecuteMethodConfig(fromAddress string, offrampProgramAddress string) ch
 			getRMNRemoteProgramAccount(offrampProgramAddress),
 			getRMNRemoteCursesLookup(offrampProgramAddress),
 			getRMNRemoteConfigLookup(offrampProgramAddress),
-			{
-				AccountLookup: &chainwriter.AccountLookup{
-					Name:       "UserAccounts",
-					Location:   "ExtraData.ExtraArgsDecoded.accounts",
-					IsWritable: chainwriter.MetaBool{BitmapLocation: "ExtraData.ExtraArgsDecoded.accountIsWritableBitmap", StartIndex: 1},
-					IsSigner:   chainwriter.MetaBool{Value: false},
-				},
-				Optional: true,
-			},
```

Comment:
Also could you add this comment here so people know that these remaining accounts are getting added in the token registry
```
// logic receiver and user defined messaging accounts are appended in the CCIPExecute args transform
```